.github/workflows/template-quality-gate-pr.yml

```diff
@@ -150,39 +150,39 @@ jobs:
             sccache-conf-path: ""
           - os: "windows"
             name: "amd64-avx2"
-            runs-on: "windows-2019"
+            runs-on: "windows-cuda-11-7"
```

Comment:
with old runner "windows-2019", it will build error due to this error: 
error LNK2019: unresolved external symbol RAND_bytes referenced in function "bool __cdecl trantor::utils::secureRandomBytes(void *,unsigned __int64)" 

-> trantor use RAND_bytes from OpenSSL maybe the default old runner missing openssl, if change to our self host runner, it will build successfully
https://github.com/janhq/cortex.llamacpp/actions/runs/10539683062/job/29203513903

Reply:
can we check for other CIs also?

Reply:
all other CIs can build successfully with self hosted runner, this issue only appear when add trantor file logger, because it use some random code from OpenSSL to create log file name.

Reply:
I'll update CI for nightly build, Release,...